src/components/panel/panel.spec.js

```diff
@@ -1475,6 +1696,29 @@ describe('$mdPanel', function() {
   }
 
   /**
+   * Attaches the panel. If a config value is passed, creates a new panelRef
+   * using $mdPanel.open(config); Otherwise, called open on the panelRef,
```

Comment:
Ahh, I agree with that call actually. 

Previously there were stand alone functions for the show, hide, close, functions etc that did this. But now they are simplified and I think this code would indeed be better removed.